include/ensmallen_bits/pso/update_policies/lbest_update.hpp

```diff
@@ -105,8 +107,7 @@ class LBestUpdate
        chi = 2.0 / std::abs(2.0 - phi - std::sqrt((phi - 4.0) * phi));
 
        // Initialize local best indices to self indices of particles.
-       localBestIndices = arma::linspace<
-           arma::Col<typename MatType::elem_type> >(0, n-1, n);
+       localBestIndices = linspace<ColType>(0, n - 1, n);
```

Comment:
These all seem to be individually accessed, so I think it is best to keep this on the CPU.  Also, I'm not sure why it was `arma::Col<typename MatType::elem_type>` to begin with---it seems like it would work just fine as `arma::uvec` (in fact that would make more sense to me).

Reply:
Okay, will leave it on the CPU.